cross-dom-bridge-erc20/index.js

```diff
@@ -38,6 +38,31 @@ const getSigners = async () => {
 }   // getSigners
 
 
+
+// The ABI fragment for the contract. We only need to know how to do two things:
+// 1. Get an account's balance
+// 2. Call the faucet to get more (only works on L1)
```

Comment:
I know its a tut but a note here that of course in prod your L1 token will not have a faucet? Feel free to ignore might be getting too into the weeds here.